components/compliance-service/habitat/default.toml

```diff
@@ -22,3 +22,4 @@ manual_polling_interval = 1440
 [agent]
 workers = 10
 buffer_size = 1000
+remote_inspec_version = "3.9.0"
```

Comment:
Typically we also specify the default in the Go code here: https://github.com/chef/automate/blob/master/api/config/compliance/config_request.go#L25

It's annoying, likely isn't strictly required anymore, but might still be good to do for consistency.

Reply:
Thank you, auto-updating the value with the most recent commit.